src/main/java/com/sleekbyte/tailor/listeners/LowerCamelCaseListener.java

```diff
@@ -28,28 +28,29 @@ public LowerCamelCaseListener(Printer printer) {
     @Override
     public void enterTopLevel(TopLevelContext topLevelCtx) {
         List<IdentifierContext> names = DeclarationListener.getVariableNames(topLevelCtx);
-        names.forEach(ctx -> verifyLowerCamelCase(Messages.VARIABLE + Messages.NAMES, ctx));
+        names.forEach(ctx -> verifyLowerCamelCase(Messages.VARIABLE + Messages.NAMES, ctx, true));
```

Comment:
Can you please add some representative functional tests to `ConstantNamingTest.swift` and  `LowerCamelCaseTest.swift`?